goldens/theme-duplicated-styles.json

```diff
@@ -0,0 +1,120 @@
+[
```

Comment:
I'm not sure I follow why we need another dedicated golden file for this. Couldn't we include the mixins twice somewhere and verify that the rules aren't duplicated?

Reply:
Isn't that we are doing as part of this PR? The golden is needed since there are actual duplicates at the time of writing. We don't want the test to fail for the current state, but rather it should fail if we regress as part of the density API or with general theme changes.

Reply:
This has changed anyway now. Marking as resolved.